### PR TITLE
Hearing Prep Help Page

### PR DESCRIPTION
**Pull request by Sjones352**
Connects #3120 and #3136 

## Acceptance Criteria
1. Verify that when the Help link is clicked from the Username Menu in the Navigation bar, the ‘Hearing Prep Help’ page is displayed.
1. Verify that the new Help page has been created with the following sections and their sub-copy (refer to mockup):

_**Screenshot1_**
screen shot 2017-10-12 at 1 37 55 pm

_**Screenshot2_**
screen shot 2017-10-12 at 1 38 20 pm

**_Training Link_ #3136 **
screen shot 2017-10-12 at 3 09 30 pm

### CODE REVIEW

File: app/views/help/hearings.html.erb [hunk 1]
```diff
@@ -1,5 +1,151 @@
 <% content_for :page_title do page_title('Hearings Help') end %>
 
 <div class="cf-app-segment cf-app-segment--alt cf-help-content">
-  <h1>Caseflow Hearings Help</h1>
+  <h1 id="#top">Welcome to the Hearings Help page!</h1>
+   <p>Here you will find <a href="#training-videos"> Training Videos</a> and <a href="#faq"> Frequently Asked Questions (FAQs)</a> for Hearing Prep, as well as links to the <%= link_to "Training Guide", asset_path("hearingprep_trainingguide.pdf"), target: "_blank" %> and the  <%= link_to "Quick Reference", asset_path("hearingprep_quickreference.pdf"), target: "_blank" %>. These items are provided to assist you as you access and use Hearing Prep. If you require further assistance after reviewing these items, please contact the Caseflow Product Support Team by phone (1-844-876-5548) or email <a href ="mailto: caseflow@va.gov">(caseflow@va.gov)</a>. We look forward to assisting you.
```
Comment (mmlumba) [inline app/views/help/hearings.html.erb]: no space between `mailto:` and the email

File: app/views/help/hearings.html.erb [hunk 2]
```diff
@@ -1,5 +1,151 @@
...
+  </p>
+
+  <h2 id="#documents-in-reader-come">16. Where do the documents in Reader come from?</h2>
+  <p>
+    The documents used in Reader come from VBMS.
+  </p>
+ 
+  <h2 id="#encounter-problems">17. What should I do if I encounter problems?</h2>
+  <p>
+    If you encounter any problems while using Hearing Prep, contact the Caseflow Product Support Team by calling 1-844-876-5548 or sending an email to <a href ="mailto: caseflow@va.gov">caseflow@va.gov</a>. They can be reached from 8:00AM to 8:00PM EST.
```
Comment (mmlumba) [inline app/views/help/hearings.html.erb]: remove spaces in the anchor tag

File: app/views/help/hearings.html.erb [hunk 3]
```diff
@@ -1,5 +1,151 @@
...
+  </p>
+
+  <h2 href="#share-feedback">18. How do I share my feedback for improving Hearing Prep?</h2>
```
Comment (mmlumba) [inline app/views/help/hearings.html.erb]: this should be an id

File: app/views/help/hearings.html.erb [hunk 4]
```diff
@@ -1,5 +1,151 @@
...
+  <p>
+    You can use the "Send feedback" link located in the dropdown menu next to your username, or the “Send feedback” link located at the bottom right-hand corner of the screen, to share your ideas for improving Hearing Prep.
+  </p>
+
+  <h2 href="#need-help">19. What if I still need help?</h2>
```
Comment (mmlumba) [inline app/views/help/hearings.html.erb]: this should be an id

File: app/views/help/hearings.html.erb [hunk 5]
```diff
@@ -1,5 +1,151 @@
...
+  <p>
+    If you require further assistance after reviewing the FAQs, Quick Reference Guide, or Training Guide, please contact the Caseflow Product Support Team by phone (1-844-876-5548) or email <a href ="mailto: caseflow@va.gov">(caseflow@va.gov)</a>. We look forward to assisting you.
+  </p>
+
 </div>
+
+<div class="cf-app-segment">
+  <div class="cf-push-left">
+    <a href ="#top">Back To Top</a>
```
Comment (mmlumba) [inline app/views/help/hearings.html.erb]: remove space

File: app/views/help/hearings.html.erb [hunk 6]
```diff
@@ -1,5 +1,151 @@
...
+  <p>
+    None of the information on the Hearing Worksheet is saved to VACOLS. Only the information changed on the Daily Docket is saved back to VACOLS.
+  </p>
+
+  <h2 id="#preliminary-mpressions-boxes">14. How are the Preliminary Impressions boxes on the Hearing Worksheet used?</h2>
+  <p>
+    The Preliminary Impressions boxes are used to indicate your first thoughts about the issues associated with the case. You may select more than one or none at all. The issue indicators are Re-Open, Dismiss, Allow, Deny, Remand, and VHA.
+  </p>
+
+  <h2 id="#case-summary-category-in-reader ">15. What does the Case Summary category in Reader represent?</h2>
```
Comment (mmlumba) [inline app/views/help/hearings.html.erb]: remove space in h2 tag

File: app/views/help/hearings.html.erb [hunk 7]
```diff
@@ -1,5 +1,151 @@
...
+    <li><a href="#preliminary-mpressions-boxes">14. How are the Preliminary Impressions boxes on the Hearing Worksheet used?</a></li>
+    <li><a href="#case-summary-category-in-reader ">15. What does the Case Summary category in Reader represent?</a></li>
+    <li><a href="#documents-in-reader-come">16. Where do the documents in Reader come from?</a></li>
+    <li><a href="#encounter-problems">17. What should I do if I encounter problems?</a></li>
+    <li><a href="#share-feedback">18. How do I share my feedback for improving Hearing Prep?</a></li>
+    <li><a href="#need-help">19. What if I still need help?</a></li>
+  </ul>
+
+  <div class="cf-help-divider"></div>
+  <h2 id="#what-is-caseflow-hearing-PREP">1. What is Caseflow Hearing Prep?</h2>
```
Comment (mmlumba) [inline app/views/help/hearings.html.erb]: The IDs shouldn't have the `#` sign in them. That's only for the `<a>` tag.

File: app/views/help/hearings.html.erb (hunk 1 above)
Comment (Sjones352) [reply app/views/help/hearings.html.erb]: Fixed

File: app/views/help/hearings.html.erb (hunk 7 above)
Comment (Sjones352) [reply app/views/help/hearings.html.erb]: Fixed

File: app/views/help/hearings.html.erb (hunk 6 above)
Comment (Sjones352) [reply app/views/help/hearings.html.erb]: Fixed

File: app/views/help/hearings.html.erb (hunk 2 above)
Comment (Sjones352) [reply app/views/help/hearings.html.erb]: Fixed

File: app/views/help/hearings.html.erb (hunk 3 above)
Comment (Sjones352) [reply app/views/help/hearings.html.erb]: Fixed

File: app/views/help/hearings.html.erb (hunk 4 above)
Comment (Sjones352) [reply app/views/help/hearings.html.erb]: Fixed

File: app/views/help/hearings.html.erb (hunk 5 above)
Comment (Sjones352) [reply app/views/help/hearings.html.erb]: Fixed

**Pull request merged**